### PR TITLE
KatyaL-hw_3

### PR DESCRIPTION
**Pull request by kate-lk**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+//     Task 1
+
+let str = "true";
```
Comment (IlyaSenkevich19) [inline script.js:3]: You can use "const" here, cause you we don't change this variable anywhere.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+let numFB = prompt("Введите число");
+
+if (numFB % 5 === 0) {
+  console.log("Fiz");
+} else if (numFB % 3 === 0) {
+  console.log("Buz");
+} else if (numFB % 5 === 0 && numFB % 3 === 0) {
+  console.log("FizBuz");
+}
```
Comment (IlyaSenkevich19) [inline script.js:51]: It is a good practice to add ELSE to the end of your condition.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+let pen = true;
+let apple = false;
+let orange = true;
+
+let shouldGoToWork = key && documents && pen && (apple || orange);
+console.log(shouldGoToWork);
+
+//     Task 5
+
+let numFB = prompt("Введите число");
```
Comment (IlyaSenkevich19) [inline script.js:43]: It is a good practice to be sure that you are working with number type, you can add '+' before 'prompt'. Also, you can add condition for letters. Just to improve it.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+//     Task 6
+
+let age = prompt("Укажите ваш возраст");
+
+if (age > 18) {
+  alert("Попей пивка");
+} else if (age >= 16 && age <= 18) {
+  akert("Можешь выкурить сигаретку, только маме не говори");
```
Comment (IlyaSenkevich19) [inline script.js:60]: Mistake, should be "alert"

File: script.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+//     Task 7
+
+let side = prompt(
+  "В какую сторону света вы хотели бы отправиться?"
+).toLocaleLowerCase();
+
+switch (side) {
+  case "юг":
```
Comment (IlyaSenkevich19) [inline script.js:72]: It will be better to move values of cases to the variable. And use them in the "switch" constuction.

File: script.js (hunk 2 above)
Comment (kate-lk) [reply script.js:53]: If only one "if", does it have to be "else"?

Comment (IlyaSenkevich19) [reply script.js:53]: No, you have to look depending on the conditions and tasks. In a case, with a prompt, we can enter data in which no IF will work, so in this situation it is important to have ELSE

**Pull request merged**